tools/windows-node-installer/test/e2e/aws_test.go

```diff
@@ -0,0 +1,27 @@
+package e2e
+
+import (
+	"github.com/openshift/windows-machine-config-operator/tools/windows-node-installer/pkg/cloudprovider"
+	"testing"
+	"time"
+)
+
+func TestE2E(t *testing.T) {
+	aws, err := cloudprovider.CloudProviderFactory("/tmp/kubeconfig", "/tmp/awscredentials", "default",
```

Comment:
You can't hard code these inputs if you want them to be used in an e2e test in CI.